app/controllers/homepage_controller.rb

```diff
@@ -141,10 +141,13 @@ def find_listings(params, listings_per_page, filter_params, includes)
       keywords: filter_params[:search],
       fields: checkboxes.concat(dropdowns).concat(numbers),
       per_page: listings_per_page,
-      page: params[:page].to_i > 0 ? params[:page].to_i : 1
```

Comment:
I didn't test this, but it looks like there's a possiblity for NullPointer if `params[:page]` is `nil`